roles/cockpit/tasks/main.yml

```diff
@@ -6,10 +6,11 @@
     - cockpit-shell
     - cockpit-bridge
     - "{{ cockpit_plugins }}"
-  when: not openshift.common.is_atomic | bool
+  when: not openshift.common.is_containerized | bool
```

Comment:
Scott and I discussed keeping containerized installs consistent.  If you perform a containerized OSE install you should get a containerized cockpit.  However, that container doesn't exist yet.  I probably won't ship this change with the upgrade I think it was just something I add because the correct cockpit RPM couldn't even be found and it was blocking my testing.